src/pythae/models/base/base_model.py

```diff
@@ -117,6 +117,26 @@ def reconstruct(self, inputs: torch.Tensor):
         """
         return self(DatasetOutput(data=inputs)).recon_x
 
+    def predict(self, inputs: BaseDataset, **kwargs) -> ModelOutput:
+        """The input data is encoded and decoded without computing loss
+        Args:
+            inputs (BaseDataset): An instance of pythae's datasets
```

Comment:
I just realized. To make it consistent with `reconstruct` and `interpolate` maybe we should expect `torch.Tensor`  type from the `inputs`? Let me know what is best suited for your use cases.

Reply:
Ya, good idea!
I have modified accordingly!

Reply:
Thanks!